### ISSUE TITLE
chore: Add word wrapping to text sections as necessary

### ISSUE THREAD

**Issue by nikiburggraf** [uw-ssec/tutorials#70]
Especially natural language outputs, needing to scroll to the side seems to (to me, at least) diminish readability.

In addition, restructuring code line lengths with a limit of 80 characters (where/when it makes sense) would likely let code snippets be displayed without needing to scroll to the side.

**Comment by lsetiawan** [comment]
Could you maybe add screenshot on this? Also, is this a nice to have or absolute must be done before demo on Tuesday?

**Comment by nikiburggraf** [comment]
This is definitely more nice-to-have than required.

It's most visible on the read-the-docs page

<img width="973" alt="Screenshot 2024-06-13 at 11 41 39 AM" src="https://github.com/uw-ssec/tutorials/assets/159066082/55f00e78-a15b-4e6b-9f61-3f9788d0a901">

As a counterexample, this is what it looks like in VSCode in a codespace 

<img width="981" alt="Screenshot 2024-06-13 at 11 43 31 AM" src="https://github.com/uw-ssec/tutorials/assets/159066082/3a5b1f8c-37e7-4535-a867-f77880726e47">

but that's with this setting ticked, and I do not remember what the default is (and I have no idea what the default would be on Nebari for the actual tutorial run at SciPy):

<img width="668" alt="Screenshot 2024-06-13 at 11 44 51 AM" src="https://github.com/uw-ssec/tutorials/assets/159066082/106c703c-b872-4fec-aa30-2753032c896f">

**Comment by lsetiawan** [comment]
Ah, gotcha, this is a jupyterbook rendering customization I think.. Thanks for providing these screenshot. I'll put it as enhancement, rather than a must to-do at this time. 😄
